src/logger/httpLoggerRules_test.go

```diff
@@ -93,13 +95,104 @@ func TestUsesAllowHttpUrlRules(t *testing.T) {
 
 // test uses copy session field rules test
 
-// func TestUsesCopySessionFieldRules(t *testing.T) {
-
-// }
+func TestUsesCopySessionFieldRules(t *testing.T) {
+	// helper for function tests
+	helper := NewTestHelper()
+	// requests used for all tests in function
+	request := NewMockRequestWithJson2()
```

Comment:
This can be `helper.MockRequestWithJson2()`. We can assume this method will be attached to the testhelper like other loggers